05-descriptive-analysis.Rmd

```diff
@@ -295,31 +297,31 @@ To calculate a population count estimate with `survey_total()`, we can run the f
 
 ```{r}
 recs_des %>%
-  summarize(survey_total(),
-            .groups = "drop")
+  summarize(survey_total(),  
+            .groups = "drop")  
 ```
 
-Note that the result from `recs_des %>% summarize(survey_total(), .groups = "drop")` is equivalent to the `survey_count()` call. However, the `survey_total()` function is called within `summarize`, where as `survey_count()`, like `dplyr::count()`, is not.
+Note that the result from `recs_des %>% summarize(survey_total(), .groups = "drop")` is equivalent to the `survey_count()` call. However, the `survey_total()` function is called within `summarize`, where as `survey_count()`, like `dplyr::count()`, is not. 
 
-The difference between `survey_total()` and `survey_count()` is more evident when specifying numeric variables to sum. Let's compute the total cost of electricity in whole dollars from variable `DOLLAREL`. We also calculate an unweighted estimate using `unweighted`.
+The difference between `survey_total()` and `survey_count()` is more evident when specifying numeric variables to sum. Let's compute the total cost of electricity in whole dollars from variable `DOLLAREL`. We also calculate an unweighted estimate using `unweighted`. 
```

Comment:
We probably need to give some examples as to why we would want to include the unweighted estimate as well.  What does this provide us in analysis?